karma.conf.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (config) {
     preprocessors: {
```

Comment:
You can remove the jenkins constant, and karma-coveralls(maybe, depends on the way lcov gets made...)